cognite/client/data_classes/data_modeling/containers.py

```diff
@@ -218,7 +218,7 @@ class ContainerProperty:
     @classmethod
     def load(cls, data: dict[str, Any]) -> ContainerProperty:
         if "type" not in data:
-            raise ValueError("Type not specified")
+            raise TypeError("Type not specified")
```

Comment:
This is not a type issue - you could argue it should be KeyError though. I think it is best to let it stay ValueError:
```suggestion
            raise ValueError("Type not specified")
```